src/Components/Notifications/NotificationItem.tsx

```diff
@@ -17,18 +17,22 @@ import { createFragmentContainer, graphql } from "react-relay"
 import styled from "styled-components"
 import { NotificationTypeLabel } from "./NotificationTypeLabel"
 import { isArtworksBasedNotification } from "./util"
+import { NotificationListMode } from "Components/Notifications/NotificationsWrapper"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
 
 const logger = createLogger("NotificationItem")
 
 interface NotificationItemProps {
   item: NotificationItem_item$data
+  mode?: NotificationListMode
 }
 
 const UNREAD_INDICATOR_SIZE = 8
 
-const NotificationItem: FC<NotificationItemProps> = ({ item }) => {
+const NotificationItem: FC<NotificationItemProps> = ({ item, mode }) => {
   const { tracking } = useNotificationsTracking()
   const { relayEnvironment } = useSystemContext()
+  const isMobile = __internal__useMatchMedia(THEME.mediaQueries.xs)
```

Comment:
Do you think there is a chance to use `Media` instead of `__internal__useMatchMedia`? If it gets to complicated, it should be ok to use `__internal__useMatchMedia`....

Reply:
@olerichter00 could you please check this commit? https://github.com/artsy/force/pull/14069/commits/66b369d84d6dbd38a1f6b22cdacbb1c638424787

Reply:
Looks good 👍